kong/api/routes/config.lua

```diff
@@ -1,5 +1,7 @@
 local declarative = require("kong.db.declarative")
+local reports = require("kong.reports")
 local kong = kong
+local reports_send = reports.send
```

Comment:
There should not be any need for localizing this function in such a cold code path? Less upvalues is better in those scenarios